package.json

```diff
@@ -24,13 +24,13 @@
   "dependencies": {
     "express": "^4.16.4",
     "react": "^16.6.3",
-    "react-dom": "^16.6.3",
+    "react-dom": "^16.6.3"
+  },
+  "devDependencies": {
     "react-redux": "^5.1.1",
     "react-router-dom": "^4.3.1",
     "redux": "^4.0.1",
-    "redux-devtools": "^3.4.2"
-  },
-  "devDependencies": {
+    "redux-devtools": "^3.4.2",
```

Comment:
I meant to move `redux-devtools` only. `react-redux`, `react-router-dom` and `redux` should stay in `dependencies`

Reply:
Done. :)